PyInstaller/hooks/hook-pycountry.py

```diff
@@ -0,0 +1,12 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
```

Comment:
Could you please add comments containing the URL of the package and about what version and what OS this hook is tested with?